Libraries/Components/Touchable/TouchableOpacity.js

```diff
@@ -261,7 +261,10 @@ class TouchableOpacity extends React.Component<Props, State> {
 
   componentDidUpdate(prevProps: Props, prevState: State) {
     this.state.pressability.configure(this._createPressabilityConfig());
-    if (this.props.disabled !== prevProps.disabled) {
+    if (
+      this.props.disabled !== prevProps.disabled ||
+      this.props.style !== prevProps.style
```

Comment:
One question, wouldn't this be `true` if inline styles were provided?

Reply:
@gabrieldonadel  didn’t got your question. can you please ellaborate/ give an example where you think this approach would be faulty

Reply:
@gabrieldonadel This would be true only if the component recieved styles which are different than the previous/existing style

Reply:
Sorry for not being clear, my question is regarding the shallow comparison between `props.style` and `prevProps.style`. If the user provides inline styles, e.g. `<TouchableOpacity style={{backgroundColor:'blue'}}/>` this if statement would always be true when a rerender occurs in the parent component, thus triggering `_opacityInactive`

![image](https://user-images.githubusercontent.com/11707729/150682206-8f219c5f-98f1-41cf-ac26-f656ab72abfa.png)

Reply:
@gabrieldonadel Okay i got it! Thanks , I will correct it

Reply:
@gabrieldonadel Now does it look good?

Reply:
I have used this code to compare function: 
https://stackoverflow.com/a/6713782/10033505